### PR TITLE
docs: Okta saml idp docs

### PR DESCRIPTION
**Pull request by hifabienne**
Adding a new guide to our docs to describe how to setup OKTA as an identity provider in ZITADEL with SAML.

### Definition of Ready

- [x] I am happy with the code
- [ ] Short description of the feature/issue is added in the pr description
- [x] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] My code has no repetitions
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

**Comment by mffap** [comment]
@hifabienne can you include references to the API, please?

File: docs/docs/guides/integrate/identity-providers/okta_saml.mdx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+<TestSetup loginscreen="your OKTA login"/>
+
+![OKTA Button](/img/guides/zitadel_login_okta.png)
+
+![OKTA Login](/img/guides/okta_login.png)
+
+:::note
+At the moment it is not possible to autofill the user data in ZITADEL with the data from OKTA.
+This means the user has to fill the information manually.
```
Comment (adlerhurst) [inline docs/docs/guides/integrate/identity-providers/okta_saml.mdx:102]: Does okta not provide any information or is an example action missing?

**Review by adlerhurst — CHANGES_REQUESTED** [review]

File: docs/docs/guides/integrate/identity-providers/okta_saml.mdx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+<Intro provider="OKTA"/>
+
+## ZITADEL Configuration
+
+### Add custom login policy
+
+<CustomLoginPolicy/>
+
+### Go to the IdP Providers Overview
```
[the remark this answers is not in the document]
Comment (hifabienne) [reply docs/docs/guides/integrate/identity-providers/okta_saml.mdx:23]: I think the most used shortcut is IdP (had a look at google), but we do have a mix in our docs

File: docs/docs/guides/integrate/identity-providers/okta_saml.mdx (hunk 1 above)
Comment (hifabienne) [reply docs/docs/guides/integrate/identity-providers/okta_saml.mdx:102]: it is not possible with the actions we currently have as far as i know

File: docs/docs/guides/integrate/identity-providers/okta_saml.mdx [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+<IDPsOverview templates="SAML SP"/>
```
[the remark this answers is not in the document]
Comment (hifabienne) [reply docs/docs/guides/integrate/identity-providers/okta_saml.mdx:25]: I did SAML SP, as this is how we have it in the console and I think that should match: 
image

File: docs/docs/guides/integrate/identity-providers/okta_saml.mdx [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+### Register a new client
+
+1. Login to your OKTA Account and go to the applications list: <OKTA-DOMAIN/admin/apps/active>
+2. Click on "Create App Integration" and choose "SAML 2.0"
+3. Give the application a name
+4. Fill the configuration as following (Replace `your-domain` and `saml-idp-id` with your data):
+ - Single sign-on URL {your-domain}/ui/login/login/externalidp/saml/acs
+ - Audience URI (SP Entity ID): {your-domain}/idps/{saml-idp-id}/saml/metadata
+ - Example redirect url for the domain `https://acme-gzoe4x.zitadel.cloud` would look like this:  `https://acme-gzoe4x.zitadel.cloud/idps/257372385775925924/saml/metadata`
```
[the remark this answers is not in the document]
Comment (hifabienne) [reply docs/docs/guides/integrate/identity-providers/okta_saml.mdx]: changed to only acme

File: docs/docs/guides/integrate/identity-providers/okta_saml.mdx [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+import Activate from './_activate.mdx';
+import TestSetup from './_test_setup.mdx';
+
+<Intro provider="OKTA"/>
+
+## ZITADEL Configuration
+
+### Add custom login policy
+
+<CustomLoginPolicy/>
```
[the remark this answers is not in the document]
Comment (hifabienne) [reply docs/docs/guides/integrate/identity-providers/okta_saml.mdx:21]: I changed it in the custom login policy.
Whenever someone touches something that is wrong we should change it. 
I already tried to find some and changed a lot in another pr, but hard to find all of them.

File: docs/docs/guides/integrate/identity-providers/okta_saml.mdx (hunk 1 above)
Comment (adlerhurst) [reply docs/docs/guides/integrate/identity-providers/okta_saml.mdx:102]: I just tested it with entra id. it should be possible: https://github.com/zitadel/actions/pull/31/files

**Review by adlerhurst — DISMISSED** [review]

**Review by adlerhurst — APPROVED** [review]

**Review by adlerhurst — CHANGES_REQUESTED** [review]
build fails

**Review by adlerhurst — APPROVED** [review]

**Review by adlerhurst — APPROVED** [review]

**Pull request merged**